src/style.css

```
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
```

move box-sizing: border-box; to `*` selector
```suggestion
* {
  box-sizing: border-box;
}

body {
  margin: 0;
}
```